### ISSUE TITLE
arkmanager always return 1

### ISSUE THREAD

**Issue by agrimal** [FezVrasta/ark-server-tools#1040]
Reading the doc, `arkmanager checkupdate` should return 1 where there is an update and 0 when there is no update. Same for `arkmanager update`. This is not the case and there is no way to know when there is an update or not. When used with systemd, the command always fails.

```
root@arkserver [~] # lsb_release -d
Description:	Debian GNU/Linux 10 (buster)
```

```bash
root@arkserver [~] # /usr/local/bin/arkmanager --version
Version: 1.6
Channel: master
Commit: 5d839a7
Release Tag: v1.6.47
Blob SHA: 17673f8d40e090d1b57e371a274b0c3781b0995d

root@arkserver [~] # /usr/local/bin/arkmanager checkupdate
Running command 'checkupdate' for instance 'main'
Querying Steam database for latest version...
Current version:  4205769 
Available version:  4205769 
Your server is up to date!

root@arkserver [~] # echo $?
1
```

```bash
root@arkserver [~] # /usr/local/bin/arkmanager update --warn --saveworld --backup --update-mods
Running command 'update' for instance 'main'
Checking for update; PID: 5021
Your server is already up to date! The most recent version is 4205769.

root@arkserver [~] # echo $?
1
```

**Comment by klightspeed** [comment]
It should work if you don't run it as root.

The `exit 1` was left over from when arkmanager would just refuse to run as root, instead of dropping privileges if run as root.

**Comment by agrimal** [comment]
I confirm the problem is solved with 1.6.48. Thanks for quick fix
